R/QFeatures-longForm.R

```diff
@@ -0,0 +1,139 @@
+##' @rdname QFeatures-class
+##'
+##' @param colvars A `character()` that selects column(s) in the `colData`.
+##'
+##' @param rowvars A `character()` with the names of the `rowData`
+##'     variables (columns) to retain in any assay.
+##'
+##' @param index When object is an instance of class `QFeatures`, a `numeric(1)`
```

Comment:
For a QFeatures, the `index` seems to allow for `numeric()`, not `numeric(1)`. Also the default is not `1L` but all assays as for `SE`s.